syntax = "proto3";

message SendMessageRequest {
  string message = 1; 
}

message SendMessageResponse {
  bool isSuccessful = 1;    
  bool isLeader = 2;  
}

service SendMessageService {
  rpc SendMessage(SendMessageRequest) returns (SendMessageResponse) {}
} 